sdcm/cluster.py

```diff
@@ -2048,6 +2048,9 @@ def node_setup(self, node, verbose=False, db_node_address=None, **kwargs):
             self.kill_stress_thread()
             return
 
+        if self.params.get('skip_loader_setup', default=False):
```

Comment:
What is this new parameter used for? And why for loader only?

Reply:
I hate to add more yaml parameters.
I thought maybe we can set an environment variable or file on the AMI and during setup we can check it. if it's there, we can skip setup.

Reply:
@bentsi If we use prepared loader AMI (contains everything: cs, collected configuration, scylla-bench, git/screen/etc), then we can skip the loader setup here. For monitor AMI, we still need to prometheus/grafana config, so we won't skip the setup as the loader.

@roydahan 
I agree with you, it's bad to introduce too many parameters to effect the logic.
At the beginning, I thought to identify the prepared AMI automatically. If c-s exists or a flag file exists (~/PREPARED-LOADER), then we will skip the loader setup, is it good?

Reply:
Yes, it's good.
I prefer a flag file.

Reply:
flag file in the AMi is a good idea  :+1:

Reply:
I just updated the loader AMI (v2) & this PR.

Tested by http://jenkins.cloudius-systems.com:8080/job/scylla-staging/job/scylla-longevity-test-dev/label=aws-eu-west1-qa-builder2/ws/latest/job.log